frontend/model/contracts/group.js

```diff
@@ -519,6 +523,46 @@ sbp('chelonia/defineContract', {
             updateCurrentDistribution({ meta, state, getters })
           }
         }
+      },
+      sideEffect ({ meta, contractID, data }, { state, getters }) {
+        if (data.updatedProperties.status === PAYMENT_COMPLETED) {
+          const { loggedIn } = sbp('state/vuex/state')
+          const payment = state.payments[data.paymentHash]
+
+          if (loggedIn.username === payment.data.toUser) {
+            sbp('gi.notifications/emit', 'PAYMENT_RECEIVED', {
+              groupID: contractID,
+              creator: meta.username,
+              paymentHash: data.paymentHash,
+              amount: getters.withGroupCurrency(payment.data.amount)
+            })
+          }
+        }
+      }
+    },
+    'gi.contracts/group/sendPaymentThankYou': {
+      validate: objectOf({
+        fromUser: string,
+        toUser: string,
+        memo: string
+      }),
+      process ({ data }, { getters }) {
+        const thankYousFrom = getters.groupThankYousFrom
+
+        const fromUser = vueFetchInitKV(thankYousFrom, data.fromUser, {})
```

Comment:
I don't think we should make variable `thankYousFrom` as it is used only once and also not complex to make it.